ui/i18n/src/main/res/values/strings.xml

```diff
@@ -640,9 +644,24 @@
     <string name="set_sleeptimer_label">Set sleep timer</string>
     <string name="disable_sleeptimer_label">Disable sleep timer</string>
     <string name="extend_sleep_timer_label">+%d min</string>
+    <plurals name="extend_sleep_timer_episodes_quantified">
+        <item quantity="one">1 episode</item>
+        <item quantity="other">%d episodes</item>
+    </plurals>
+    <string name="sleep_timer_without_continuous_playback_title">Disable continuous playback</string>
+    <string name="sleep_timer_without_continuous_playback_message">You can simply disable continuous playback to automatically stop after an episode, there is no need to use the sleep timer for that.\n\nIf you actually want that to happen only at specific times you can set up a sleep timer.</string>
+    <string name="sleep_timer_without_continuous_playback_disable_playback">Disable continuous playback</string>
+    <string name="sleep_timer_without_continuous_playback_proceed_with_timer">Use sleep timer</string>
     <string name="sleep_timer_always">Always</string>
     <string name="sleep_timer_label">Sleep timer</string>
     <string name="time_dialog_invalid_input">Invalid input, time has to be an integer</string>
+    <string name="timer_exceeds_remaining_time_while_continuous_playback_disabled">Continuous playback is disabled and the current episode has only %1$d minutes left</string>
```

Comment:
This should be plural. Some languages have different forms for 11, 21, 31, 41 minutes (not just a special case for 1 like in English)